instrumentation/servlet/servlet-5.0/javaagent/servlet-5.0-javaagent.gradle

```diff
@@ -0,0 +1,15 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "jakarta.servlet"
+    module = 'jakarta.servlet-api'
+    versions = "[5.0.0,)"
```

Comment:
Can we `assertInverse`? Since package name changed assuming it's safe so good to verify.

Reply:
Done.